### DISCUSSION TITLE
Welcome to Flutter-WhatsApp-Clone-Clean-Architecture Discussions!

### DISCUSSION THREAD

**Discussion by mogohary99** [mogohary99/Flutter-WhatsApp-Clone-Clean-Architecture#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eduvhc** [comment]
I'm learning clean architecture. 
It's a good practice to create a dependency in `Stream` types in repositories? 
Like implementing a Stream in the data source, suppose I want a data source implementation with SQLite? How do I implement a get{something}? 

Greeting from Portugal and thanks for the awesome project.
